nuget/MacAppCenter.nuspec

```diff
@@ -16,7 +16,7 @@
     <iconUrl>https://aka.ms/k76877</iconUrl>
     <dependencies>
         <!-- Use dependency groups for iOS and Android so that additional dependencies aren't added to them -->
-        <group targetFramework="MonoAndroid403">
+        <group targetFramework="MonoAndroid50">
```

Comment:
Was this reference target tested with nugets in the demo?

Reply:
I checked on all our test apps and also checked it on an empty project. In general, all works well.